spec/javascripts/surveys-spec.js

```diff
@@ -90,25 +87,109 @@ describe('Surveys', function () {
       expect(GOVUK.cookie(surveys.surveySeenCookieName(defaultSurvey))).toBe('1')
     })
 
-    describe("for a 'url' survey", function () {
-      it('links to the url for a smartsurvey survey with a completion redirect query parameter', function () {
-        surveys.displaySurvey(urlSurvey)
+    describe('for a url survey', function () {
```

Comment:
I think we should drop this describe block (but keep the tests).  We do the `{{currentPath}}` stuff for all survey types, so singling it out as something for url surveys only will give future developers the wrong idea about what we do.  Perhaps to be explicit we should duplicate the specs contained here and add them to `for an email survey` describe block as well?